neo-go.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Neo Go node
+Requires=network.target
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/neo-go node --mainnet --config-path=/etc/neo-go
```

Comment:
I think we can we make it a little better by templating this service file and expanding paths to what is specified in the `Makefile`. You can write something like
```
ExecStart=$$BINDIR$$/neo-go node --mainnet --config-path=/etc/neo-go
```
here and then in the Makefile do
```
neo-go.service: neo-go.service.template
    sed -r 's,\$\$BINDIR\$\$,$(BINDIR)' $< >$@

install: build neo-go.service
...
```
The same applies to `config-path` and `WorkingDirectory`.

Reply:
done